src/components/Layout.tsx

```diff
@@ -1,4602 +1,18 @@
-        }
-        .fa-tint:before {
-          content: '\f043';
-        }
-        .fa-tint-slash:before {
-          content: '\f5c7';
-        }
-        .fa-tired:before {
-          content: '\f5c8';
-        }
-        .fa-toggle-off:before {
-          content: '\f204';
-        }
-        .fa-toggle-on:before {
-          content: '\f205';
-        }
-        .fa-toilet:before {
-          content: '\f7d8';
-        }
-        .fa-toilet-paper:before {
-          content: '\f71e';
-        }
-        .fa-toilet-paper-slash:before {
-          content: '\f972';
-        }
-        .fa-toolbox:before {
-          content: '\f552';
-        }
-        .fa-tools:before {
-          content: '\f7d9';
-        }
-        .fa-tooth:before {
-          content: '\f5c9';
-        }
-        .fa-torah:before {
-          content: '\f6a0';
-        }
-        .fa-torii-gate:before {
-          content: '\f6a1';
-        }
-        .fa-tractor:before {
-          content: '\f722';
-        }
-        .fa-trade-federation:before {
-          content: '\f513';
-        }
-        .fa-trademark:before {
-          content: '\f25c';
-        }
-        .fa-traffic-light:before {
-          content: '\f637';
-        }
-        .fa-trailer:before {
-          content: '\f941';
-        }
-        .fa-train:before {
-          content: '\f238';
-        }
-        .fa-tram:before {
-          content: '\f7da';
-        }
-        .fa-transgender:before {
-          content: '\f224';
-        }
-        .fa-transgender-alt:before {
-          content: '\f225';
-        }
-        .fa-trash:before {
-          content: '\f1f8';
-        }
-        .fa-trash-alt:before {
-          content: '\f2ed';
-        }
-        .fa-trash-restore:before {
-          content: '\f829';
-        }
-        .fa-trash-restore-alt:before {
-          content: '\f82a';
-        }
-        .fa-tree:before {
-          content: '\f1bb';
-        }
-        .fa-trello:before {
-          content: '\f181';
-        }
-        .fa-tripadvisor:before {
-          content: '\f262';
-        }
-        .fa-trophy:before {
-          content: '\f091';
-        }
-        .fa-truck:before {
-          content: '\f0d1';
-        }
-        .fa-truck-loading:before {
-          content: '\f4de';
-        }
-        .fa-truck-monster:before {
-          content: '\f63b';
-        }
-        .fa-truck-moving:before {
-          content: '\f4df';
-        }
-        .fa-truck-pickup:before {
-          content: '\f63c';
-        }
-        .fa-tshirt:before {
-          content: '\f553';
-        }
-        .fa-tty:before {
-          content: '\f1e4';
-        }
-        .fa-tumblr:before {
-          content: '\f173';
-        }
-        .fa-tumblr-square:before {
-          content: '\f174';
-        }
-        .fa-tv:before {
-          content: '\f26c';
-        }
-        .fa-twitch:before {
-          content: '\f1e8';
-        }
-        .fa-twitter:before {
```

Comment:
After further testing, I found that the error disappeared when I removed the css of the SNS icons.
I haven't followed the detailed cause-and-effect relationship, but as I mentioned earlier, I wanted to slim down the CSS, so I'm going to remove the font awesome itself.